components/tooltip/index.tsx

```diff
@@ -81,8 +81,8 @@ const splitObject = (obj: any, keys: string[]) => {
   return { picked, omitted };
 };
 
-class Tooltip extends React.Component<TooltipProps, any> {
-  static defaultProps = {
+class Tooltip extends React.Component<TooltipProps & React.HTMLAttributes<HTMLElement>, any> {
+    static defaultProps = {
```

Comment:
Unnecessary indent.

Reply:
Thanks, I have fixed this in https://github.com/ant-design/ant-design/pull/16195/commits/fec88af3d342d719c2c4c0f7210702e7d1d015e6